force-app/main/default/lwc/note/note.js

```diff
@@ -39,57 +33,30 @@ export default class Note extends LightningElement {
         }catch (error){
             console.error('Error loading data', error);
         }
-        console.log('Updated savedData: ', this.savedData);
+        // console.log('Updated savedData: ', this.savedData);
     }
 
     connectedCallback(){
         this.loadSavedData();
         console.log('Work');
     }
 
-
-    //  loadSavedData() {
-         
-    //     console.log('Loading data from server...');
-    //     getSavedNotes()
-         
-
-    //         .then((result)=>{
-    //             this.savedData = result;
-    //             console.log('Saved Data: ',  this.savedData);
-    //         })
-    //         .catch((error)=> {
-    //         console.error('Error loading data', error);
-    //         });
-    //     console.log('Updated savedData: ', this.savedData);
-    // }
-    
-    //to do does not update when added
     async handleAdd(){
         const result = await MyModal.open({
             size: 'Small',
             label: 'Note',
             description: 'This is a modal popup'
         });
-
+        console.log('work' +  result);
         if(result) {
             try{
                 const savedNoteId = await saveData({
                     label: result.label,
                     description: result.description
                 });
 
-                console.log('Saved note ID:', savedNoteId);
+                // console.log('Saved note ID:', savedNoteId);
                     if(savedNoteId){
```

Comment:
you don't need this condition because if returns always successfully inserted record if there is no error, otherwise you catch an error below in catch block